client/src/outside/components/ActivityListItem.vue

```diff
@@ -1,8 +1,9 @@
 <template lang="pug">
   div(:id="activityId")
     div(class="activity-list-header columns")
-      div(class="header-column is-10 column")
-        h3(:title="activityId") {{ activityName }}
+      div(class="header-column is-10 column")    
+        h3(:title="activityId") 
+             router-link(:to="`/ehr/patient/demographics?readonly=true`") {{ activityName }}
```

Comment:
Initializing the "readonly" state via a query parameter introduces questions and these show a need to rethink how the application handles state.  
Q: it's so weak to be hard coding paths into the ehr pages. The demographics page may not always be the desired first page.
Q: when is the query parameter "readonly=true" consumed?  Normally this is in App.vue. But now?
Q: what happens if an instructor in the readonly mode does a browser refresh? The query parameter will not be there on refresh.
The code in EhrContextBanner does answer the above questions but it seems weak to be using that component to manage this state.  We do it because that is one component that is part of all EHR internal pages. So it works.  It is also why it's important for that component to perform "on leave" clean up.  

No need to take any action for this PR.  I've made a PR to track the questions.  https://github.com/BCcampus/edehr/issues/646